CHANGES.md

```diff
@@ -1,3 +1,129 @@
+### v0.15.1
+
+-  **TreeController**
+
+   Updates to the new version.
+
+-  **Component**
+
+   Makes creating Component’s templateObjects lazy and on-demand.
+   Reduces memory consumption.
+
+-  **Core**
+
+   Fixes super infinite loop in observed getter/setters.
+
+-  **DocumentResources**
+
+   Fix a bug affecting Firefox that was allowing components to be entering the document before its CSS stylesheets were loaded.
+
+-  **EventManager**
+
+   Makes preparedForActivationEvents public.
+   Fixes a case where the first touch in an app would be lost.
+   Adds support of PointerEvents.
+   Adds support of devices with multiples pointer types. (mouse/touch).
+   PrepareForActivationEvents is called when a pointing device (mouse/touch) is moved over a component’s element.
+   Blocks emulated mouse events.
+   Adds an option `blocksEmulatedEvents` to disable the filtering of emulated mouse events.
```

Comment:
Adds *a `blocksEmulatedEvents` option* to disable the filtering of emulated mouse events.